src/sentry/web/frontend/idp_email_verification.py

```diff
@@ -1,18 +1,25 @@
-from django.http.response import HttpResponseNotFound, HttpResponseRedirect
+from typing import Any, Mapping
+
+from django.http.response import HttpResponse
 from rest_framework.request import Request
 from rest_framework.response import Response
 
-from sentry.auth.idpmigration import get_redis_key, verify_account
+from sentry.auth.idpmigration import verify_account
 from sentry.utils.auth import get_login_url
+from sentry.web.helpers import render_to_response
+
+
+def _respond(
```

Comment:
is this function here for typing purposes? it doesn't add much IMO, setting a 302 is fine to do in two places.

Reply:
i made a private function for this since auth helper used it for the exact same purpose, looking at it, it doesn't add much and could be replaced with a direct render_to_response() call.